docs/tutorial/using-checkbox/manifest.rst

```diff
@@ -0,0 +1,299 @@
+can select ``n`` when Checkbox asks if you want to upload the test results).
+
+You can see the ``has_touchscreen`` key from the Hardware Manifest job is
+now set to ``False``:
+
+.. code-block:: none
+    :emphasize-lines: 11
+
+    =========[ Running job 2 / 3. Estimated time left (at least): 0:00:02 ]=========
+    -----------------------------[ Hardware Manifest ]------------------------------
+    ID: com.canonical.plainbox::manifest
+    Category: com.canonical.plainbox::uncategorised
+    ... 8< -------------------------------------------------------------------------
+    ns: com.canonical.certification
+    name: checkbox-provider-base
+    has_audio_playback: 
+    (...)
+    has_touchpad: 
+    has_touchscreen: False
+    (...)
+    has_wlan_adapter: 
+    has_wwan_module:
+     
+    (...)
+    
+    ------------------------------------------------------------------------- >8 ---
+    Outcome: job passed
+
+and as a result, the job that depends on it is skipped:
+
+.. code-block:: none
+    :emphasize-lines: 5-6
+
+    ---------------------------[ A job using a manifest ]---------------------------
+    ID: com.canonical.certification::tutorial/manifest
+    Category: com.canonical.certification::tutorial
+    Job cannot be started because:
+     - resource expression "manifest.has_touchscreen == 'True'" evaluates to false
+    Outcome: job cannot be started
+
+Modifying the manifest information manually
+===========================================
+
+Let's check the content of the manifest file:
+
+.. code-block:: none
+
+    $ cat /var/tmp/checkbox-ng/machine-manifest.json
+	
+    {
+      "com.canonical.certification::has_touchscreen": false
+    }
+
+As you can see, it's a JSON file that stores the value for each of the entries
+(only one in our case :)) we need to run our test plan.
+
+Using your favorite text editor, open the file, replace ``false`` with
+``true`` and save your modifications.
+
+Run Checkbox again, following the same steps we have done so far, and you will
+see in the System Manifest screen that ``Touchscreen`` is now set to ``Yes``.
+
+It can be very useful to provide pre-filled manifest files, especially when
+you want to automate Checkbox testing.
+
+Skipping the System Manifest screen
+===================================
+
+So far, every time you started Checkbox to run this test plan, you were
+greeted with the System Manifest screen to confirm the manifest information
+was correct prior to starting the test.
+
+Let's assume it is. Yes, our hypothetical test device has a touch screen,
+and we want to execute the test plan right away.
+
+First of all, make sure the information in the manifest file is correct:
+
+.. code-block:: none
+
+    $ cat /var/tmp/checkbox-ng/machine-manifest.json
+	
+    {
+      "com.canonical.certification::has_touchscreen": true
+    }
+
+Next, create a launcher file named ``auto-manifest`` with the following content:
```

Comment:
Should the file be placed under the `checkbox-ng` directory?

Reply:
It can be placed anywhere, that's why I didn't mention. Should I explicitly mention it can be put anywhere?

Reply:
I asked because in the command `$ checkbox-cli launcher auto-manifest`, I wondered whether "auto-manifest" was a command or a file path. If it can be placed in any directory, then we can show the example as e.g. `$ checkbox-cli launcher <path>/auto-manifest`?

Reply:
Fixed in [6bbfca4](https://github.com/canonical/checkbox/pull/736/commits/6bbfca44e8fb70498d8c1c78e25e01802740cdbb)